components/users/ActiveUsers.tsx

```diff
@@ -0,0 +1,34 @@
+import {useOthers, useSelf} from "@/liveblocks.config";
+import {Avatar} from "@/components/users/Avatar";
+
+import styles from './index.module.css'
+import {generateRandomName} from "@/lib/utils";
+import {useMemo} from "react";
+
+const ActiveUsers = () => {
+    const users = useOthers();
+    const currentUser = useSelf();
+    const hasMoreUsers = users.length > 3;
+
+    const memoizedUsers = useMemo(() => {
+        return (
+            <div className="flex items-center justify-center gap-1 py-2">
+                <div className="flex pl-3">
+                    {currentUser && (
+                        <Avatar name="You" otherStyles="border-[3px] border-primary-green"/>
```

Comment:
reicht nicht vielleicht border-2 aus?
```suggestion
                        <Avatar name="You" otherStyles="border-2 border-primary-green"/>
```